### PR TITLE
Ui/use real backend

### PR DESCRIPTION
**Pull request by nielsboecker**
**Note: The UI/BE/FHIR-mapping PR should be accepted first, then the diff for this one will be much smaller.**
- several refactorings and renamings
- switch from using static JSONs in the front-end to querying the back-end, which will query the FHIR server, decode and map the data, and return it to the front-end
- Introduce Context in front-end for global state

### CODE REVIEW

**Comment by nielsboecker** [comment]
To the reviewers: Sorry, this one is kinda big, because I did a big refactoring which touched pretty much every single file. If you click on "Changed files" right here in the PR in GitHub, it will show renamings only very small, and helps you to focus on code changes.

File: HoloRepositoryUI/client/src/services/holoRepositoryServerService.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import serverAxios, { routes } from "./holoRepositoryServerAxios";
+import {
+  IPatient,
+  IPractitioner,
+  IHologram,
+  IImagingStudy,
+  IPipeline
```
Comment (fanbomeng97) [inline HoloRepositoryUI/client/src/services/holoRepositoryServerService.ts:7]: It's fine, but just curious why there is an "I" in front of the class name? any special purpose?

**Review by fanbomeng97 — APPROVED** [review]
Leave a question here, the code is fine, and works well.

File: HoloRepositoryUI/client/src/services/holoRepositoryServerService.ts (hunk 1 above)
Comment (nielsboecker) [reply HoloRepositoryUI/client/src/services/holoRepositoryServerService.ts:7]: It's just a naming convention for TypeScript interfaces (so the I is for interface). There are different opinions on whether or not to do this, but I think as long as we do it consistently it is fine.

Comment (fanbomeng97) [reply HoloRepositoryUI/client/src/services/holoRepositoryServerService.ts:7]: OK, it's fine, I have done the review for the code, will merge it, nice work!

**Pull request merged**